index.html

```diff
@@ -1969,10 +2026,19 @@ <h3>Securing Verifiable Credentials</h3>
         <dl>
           <dt><var>proof</var></dt>
           <dd>
-One or more cryptographic proofs that can be used to detect tampering and verify
-the authorship of a <a>verifiable credential</a> or <a>verifiable
-presentation</a>. The specific method used for an <a>embedded proof</a> MUST be
-identified using the <code>type</code> <a>property</a>.
+            <p>
+              One or more cryptographic proofs that can be used to detect tampering and verify
+              the authorship of a <a>verifiable credential</a> or a <a>verifiable
+              presentation</a>. Each proof is a separate <a>named graph</a>
+              (referred to as a <dfn>proof graph</dfn>) containing a single proof.
+              The specific method used for an <a>embedded proof</a> MUST be
+              identified using the <code>type</code> <a>property</a>.
+            </p>
+            <p>
+              A proof for a <a>verifiable credential</a> proves the validity of all <a>claims</a>
+              included in the corresponding <a>verifiable credential graph</a>. See <a href="#presentations-0"></a>
+              for the case when the property is used for a <a>verifiable presentation</a>.
```

Comment:
We shouldn't use the term "validity" here, it leads to confusion. I suggest:

```suggestion
              A proof for a <a>verifiable credential</a> covers all <a>claims</a>
              included in the corresponding <a>verifiable credential graph</a>. See <a href="#presentations-0"></a>
              for the case when the property is used for a <a>verifiable presentation</a>.
```